src/main/webapp/app/shared/util/tab.constants.ts

```diff
@@ -0,0 +1,29 @@
+import { ITabInfo } from 'app/shared/components/customTab/custom-tab';
+
+export const eventTabList: ITabInfo[] = [
+  {
+    tabName: 'Details',
+    tabTranslateKey: 'clubmanagementApp.event.tab',
+    tabRoute: '/'
+  },
+  {
+    tabName: 'Checklist',
+    tabTranslateKey: 'clubmanagementApp.eventChecklist.tab',
+    tabRoute: '/checklist'
+  },
+  {
+    tabName: 'Budget',
+    tabTranslateKey: 'clubmanagementApp.eventBudget.tab',
+    tabRoute: '/budget'
+  },
+  {
+    tabName: 'Attendees',
+    tabTranslateKey: 'clubmanagementApp.eventAttendee.tab',
+    tabRoute: '/attendees'
+  },
+  {
+    tabName: 'Crews',
+    tabTranslateKey: 'clubmanagementApp.eventCrew.tab',
+    tabRoute: '/crews'
```

Comment:
why the tabRoute not with prefix "/event/"  ❓ 

and this tabRoute should synchronize with the actual route of the page
there are existing pages either follow the one 
or we need to change that Route path also

Reply:
The tabRoute prefix is resolved in 3348d0f8db46f3f45240588c3776cdd7a27b074e

For the existing pages, may I know where can I refer to?